core/src/odbc_uri.rs

```diff
@@ -45,35 +46,56 @@ lazy_static! {
 }
 
 #[derive(Debug, PartialEq, Eq)]
-pub struct ODBCUri<'a>(HashMap<String, &'a str>);
+pub struct ODBCUri(HashMap<String, String>);
 
-impl<'a> std::ops::Deref for ODBCUri<'a> {
-    type Target = HashMap<String, &'a str>;
+impl std::ops::Deref for ODBCUri {
+    type Target = HashMap<String, String>;
 
     fn deref(&self) -> &Self::Target {
         &self.0
     }
 }
 
-impl<'a> ODBCUri<'a> {
-    pub fn new(odbc_uri: &'a str) -> Result<ODBCUri<'a>> {
+fn transform_keyword(keyword: &str) -> String {
+    match keyword {
+        UID => USER.to_string(),
+        USER => USER.to_string(),
+        PWD => PASSWORD.to_string(),
+        PASSWORD => PASSWORD.to_string(),
```

Comment:
Looks like we don't need `USER` or `PASSWORD` here since the `_` branch will handle these the same way?

Reply:
True true, but I wanted the transformation to be explicit. I can remove if you don't think it helps.

Reply:
Thanks for the explanation.  Let's leave it in 👍